### ISSUE TITLE
ExpressionBodySyntax false positive on overridden methods

### ISSUE THREAD

**Issue by personshelldon** [arturbosch/detekt#1794]
I think, that there is a false positive result for `ExpressionBodySyntax` in code like this:
```
override fun getColumnSize(columnIndex: Int): Int {
            return layoutManager.getColumnWidth(columnIndex)
        }
```
Bacause `getColumnSize` is declared in the interface and can have many implementations, I think the `ExpressionBodySyntax` has to ignore such cases.

Also, there is a parameter in method, that is not very good for converting it to `ExpressionBodySyntax`

**Android Studio:** 3.1.4
**Test version:** RC16

**Comment by schalkms** [comment]
> Bacause getColumnSize is declared in the interface and can have many implementations, I think the ExpressionBodySyntax has to ignore such cases.

That's no reason to prevent the implementation of the function to use the shorthand syntax. The following code is perfectly valid in Kotlin.

```Kotlin
override fun getColumnSize(columnIndex: Int) = layoutManager.getColumnWidth(columnIndex)
```

> Also, there is a parameter in method, that is not very good for converting it to ExpressionBodySyntax

I don't understand. The parameter doesn't matter for the `ExpressionBodySyntax` rule.
